app/templates/components/forms/session-section.hbs

```diff
@@ -43,7 +43,8 @@
           @hint={{t "Select an image"}}
           @maxSizeInKb={{this.settings.logoSize}} />
       {{/if}}
-      {{#if (and (eq field.type 'file') (eq field.fieldIdentifier 'slides'))}}
+      {{#if (eq field.type 'file')}}
+        {{#if (or (eq field.fieldIdentifier 'slides') (eq field.fieldIdentifier 'slidesUrl'))}}
```

Comment:
don't add extra if, use it in me statement

Reply:
me??

Reply:
Sorry, means use in one line.